force-app/main/default/lwc/nksEmbeddedVideoPlayer/nksEmbeddedVideoPlayer.js

```diff
@@ -117,18 +119,18 @@ export default class NksEmbeddedVideoPlayer extends LightningElement {
     attachEventListener() {
         const video = this.template.querySelector('video');
         if (video) {
+            this.videoEventListenerAttached = true;
             video.addEventListener('play', () => {
-                this.addViewCount();
+                if (!video.paused) {
+                    this.addViewCount();
+                }
             });
```

Comment:
Så ikke dette forrige gang, men kommer ikke detta til å adde fette mange viewCounts hvis æ spamme pause/play? Vi burde vel ha en maks grense på 1 for hver pers. Og hvorfor sjekker du video.paused her? Hvis eventen "play" kjøres, er ikke videon playing da?

Reply:
Video.paused: Fordi den også trigger på om man trykker pause knappen (det er i prinsippet samme knapp som play knappen, bare endrer logo).
Den lytter kun én gang, somehow. Jeg har lagt til {once : true} som argument nå for å være på den sikre siden :)